airflow/providers/databricks/operators/databricks_sql.py

```diff
@@ -70,14 +70,11 @@ class DatabricksSqlOperator(SQLExecuteQueryOperator):
     :param csv_params: parameters that will be passed to the ``csv.DictWriter`` class used to write CSV data.
     """
 
-    template_fields: Sequence[str] = (
-        "sql",
-        "_output_path",
-        "schema",
-        "catalog",
-        "http_headers",
-        "databricks_conn_id",
+    template_fields: Sequence[str] = tuple(
+        {"_output_path", "schema", "catalog", "http_headers", "databricks_conn_id"}
```

Comment:
@alexott I find it really confusing that this operator has `databricks_conn_id` from this class and also `conn_id` from parent class.

Something worth addressing in a followup PR